### PR TITLE
feature: add vulcansql for community integrations

### PR DESCRIPTION
**Pull request by cyyeh**
Add VulcanSQL in the community integrations

### CODE REVIEW

**Comment by cyyeh** [comment]
@jkaflik
Hi, I am one of the core members of VulcanSQL, a Data API framework for data applications. Clickhouse people are so nice and give us lots of support in this tweet: https://twitter.com/vulcansql/status/1677836987007664128! Thank you!

Also thanks for reviewing this PR to make VulcanSQL appear on ClickHouse documentation!

**Comment by robot-ch-test-poll1** [comment]
This is an automated comment for commit ab201876cb6e4e517ff129ee2040b4078473d2c2 with description of existing statuses. It's updated for the latest CI running
The full report is available [here](https://s3.amazonaws.com/clickhouse-test-reports/1332/ab201876cb6e4e517ff129ee2040b4078473d2c2/ci_running.html)
The overall status of the commit is 🟢 success

<table><thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody><tr><td>Docs Check</td><td>Builds and tests the documentation</td><td>🟢 success</td></tr>
<tr><td>Vercel</td><td>There's no description for the check yet, please add it to tests/ci/ci_config.py:CHECK_DESCRIPTIONS</td><td>🟢 success</td></tr>
</table>

File: docs/en/integrations/index.mdx [hunk 1]
```diff
@@ -188,4 +188,5 @@ We are actively compiling this list of ClickHouse integrations below, so it's no
...
 |Visual Studio Client|<img src={require('./images/logos/logo_vs.png').default} class="image" alt="VS logo" style={{width: '3rem', 'background-color': 'transparent'}}/>|Language client|Visual studio lightweight client|[Marketplace](https://marketplace.visualstudio.com/items?itemName=fanruten.clickhouse-light)|
 |Easypanel|<img src={require('./images/logos/logo-easypanel.png').default} class="image" alt="Easypanel logo" style={{width: '3rem', 'background-color': 'transparent'}} />|Deployment method|It's a modern server control panel. You can use it to deploy ClickHouse on your own server.|[Website](https://easypanel.io), <br/>[Documentation](docs/en/integrations/deployment/easypanel/index.md)|
+|VulcanSQL|<img src={require('./images/logos/logo-vulcansql.png').default} class="image" alt="VulcanSQL logo" style={{width: '3rem', 'background-color': 'transparent'}} />|Data API Framework|It's a Data API Framework for data applications that helps data folks create and share data APIs faster. It turns your SQL templates into data APIs. No backend skills required.|[Website](https://vulcansql.com/)|
```
Comment (mshustov) [inline docs/en/integrations/index.mdx:191]: SHouldn't the link navigate to the ClickHouse connector? https://vulcansql.com/docs/connect/clickhouse

Comment (cyyeh) [reply docs/en/integrations/index.mdx]: I fixed it. Thanks

**Pull request merged**